### DISCUSSION TITLE
Give option to permanently hide "Overwhelmed by notifications banner"

### DISCUSSION THREAD

**Discussion by imciner2** [community/community#61609] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The "Overwhelmed by notifications" banner periodically appears in my notifications inbox, but I don't care about it - I purposely subscribe to the repository notifications I want, so why would I want to change that? . This banner is annoying. Please add a way to permanently hide this banner and make it never appear in my inbox again instead of just the "X" button that only makes it disappear temporarily. I never want to interact with that banner!!!!

**Comment by passcod** [comment]
Second this. In the entire history of me seeing this notification (sometimes for laughably low notification amounts), I have _never_ felt overwhelmed.
